jedi_vim.py

```diff
@@ -254,7 +254,7 @@ def goto(mode="goto", no_output=False):
                         tagname = d.name
                         while vim_eval('taglist("^%s$")' % tagname) != []:
                             tagname = d.name + ' ' + ''.join(
-                                [random.choice(string.lowercase)
+                                [random.choice(string.lower)
```

Comment:
Yeah, the fix is not correct either.

`string` is imported as a module, and `string.lowercase` exists in Python 2: `'abcdefghijklmnopqrstuvwxyz'`.

`string.ascii_lowercase` should be used for both Python 2/3.